src/content/docs/tutorial-reporting-data/capture-web-telemetry.mdx

```diff
@@ -0,0 +1,354 @@
+---
+title: "Capture the right web telemetry" 
+metaDescription: "This guide helps you set up the best telemetry possible to describe the runtime operation of your web services."
+redirects:
+  - /docs/new-relic-solutions/observability-maturity/operational-efficiency/sc-implementation-guide
+  - /docs/new-relic-solutions/observability-maturity/operational-efficiency/service-characterization-optimize-web-telemetry-guide/
+---
+
+import omaOeScBrowserErrors from 'images/oma-oe-sc_screenshot-browser-errors.webp'
+import omaOeScBrowserErrorLog from 'images/oma-oe-sc_screenshot-browser-errorlog.webp'
+import omaOeScBrowserGroupedPageViews from 'images/oma-oe-sc_screenshot-poor-pageview-grouping.webp'
+import omaOeScBrowserGroupedAjaxRequests from 'images/oma-oe-sc_screenshot-poor-ajax-grouping.webp'
+
+One of the most important things you can do to maintain a high quality production environment is to make sure you have the web telemetry you need to detect and resolve poor user experience. This guide will show you how to check that you're getting the right data and that your browser monitoring is configured for you to make the most of it. We'll help you ensure that you're:   
```

Comment:
```suggestion
One of the most important things you can do to maintain a high quality production environment is to make sure you have the web telemetry you need to detect and resolve poor user experience. This guide will show you how to check that you're getting the right data and that your browser monitoring is optimally configured. We'll help you ensure that you're:   
```

Reply:
Felt a bit long and clunky. Feel free to rewrite or ignore this one

Reply:
Reworded to hopefully make it less clunky.